chain/epoch_manager/src/lib.rs

```diff
@@ -71,12 +71,15 @@ pub struct EpochManager {
     epoch_info_aggregator: Option<EpochInfoAggregator>,
     /// Largest final height. Monotonically increasing.
     largest_final_height: BlockHeight,
+    /// Number of additional epochs to keep in memory.
+    pub additional_epochs_to_keep: u64,
```

Comment:
Not a fan of this being a *single* `pub` field. As a rule-of-thumb, types where all fields are private are good -- they are an abstraction boundary. Types where all fields are public are also good -- they are typically just PODs. Types where *some* fields are public are often a smell, as the abstraction is inconsistently enforced. So, adding the first pub field to a type which has a bunch of non-pub fields is not something which should be done lightly, seems better to hide this behind a getter just in case.